assets/helpers/abtest.js

```diff
@@ -165,6 +177,26 @@ export const getVariantsAsString = (participation: Participations): string => {
   return variants.join('; ');
 };
 
+export const trackOphan = (
+  testId: TestId,
+  variant: string,
+  complete?: boolean = false,
+  campaignCodes?: string[] = []): void => {
+
+  const payload: OphanABPayload = {};
+
+  payload[testId] = {
+    variantName: variant,
+    complete,
+    campaignCodes,
+  };
```

Comment:
yes, this new annotation was introduced in es6 as a new way of defined [computed property names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names)